src/darwin/Framework/CHIP/MTRDevice.h

```diff
@@ -98,7 +98,9 @@ typedef NS_ENUM(NSUInteger, MTRDeviceState) {
 /**
  * Read attribute in a designated attribute path
  *
- * TODO: Need to document that this returns "the system's best guess" of attribute values.
+ * TODO: Need to fully document that this returns "the system's best guess" of attribute values.
+ * Note: This method will return nil for both attributes that are not yet subscribed to (which is done automatically),
```

Comment:
I don't see any changes to actual behavior here...

Reply:
"* Note: This method will return nil for both attributes that are not yet ..."

Reply:
Yes, the _documentation_ is changing.  But the behavior is not changing.  There are no changes to MTRDevice.mm here.